docs/conf.py

```diff
@@ -120,74 +121,6 @@
 autosummary_generate = True
 napolean_use_rtype = False
 
-# -- Options for nbsphinx -----------------------------------------------------
-
-# Execute notebooks before conversion: 'always', 'never', 'auto' (default)
-# We execute all notebooks, exclude the slow ones using 'exclude_patterns'
-nbsphinx_execute = 'always'
```

Comment:
We still have notebooks, until we transition them. Should we drop these options already?

Reply:
I think I understand, myst-nb will execute both .md and .ipynb files.

Reply:
Right - nbsphinx and myst-nb cannot be used together. This PR removes nbsphinx & changes it so that myst-nb is in charge of *all* notebooks, so the nbsphinx settings are no longer relevant.